backend/src/main/java/co/inventorsoft/scripty/exception/ScriptyUncheckedException.java

```diff
@@ -0,0 +1,25 @@
+package co.inventorsoft.scripty.exception;
+
+/**
+ * ScriptyUncheckedException 
+ * All project's custom unchecked exceptions should use or extend it
+ */
+public class ScriptyUncheckedException extends RuntimeException {
```

Comment:
Let's name it `ApplicationException` and add a status code field to it. It would be useful to render response based on that since we're working in web based environment